package.json

```diff
@@ -849,6 +849,11 @@
                     "type": "webview",
                     "id": "julia-documentation",
                     "name": "Documentation"
+                },
+                {
+                    "type": "webview",
+                    "id": "julia-plot-navigator",
+                    "name": "Plot Navigator"
```

Comment:
It might make sense to add a `"when": "hasJuliaREPL"` here -- the navigator is basically useless when there's no running Julia session, right?

Reply:
Well, one can open a REPL, plot some things, kill the REPL and still navigate around in the plot panel, right? I.e. plots outlive the REPL.

Reply:
True that.